### PR TITLE
Add keyboard shortcuts in reader for categories

### LINKED ISSUE #8519: Add keyboard short-cut to add tags
https://github.com/department-of-veterans-affairs/appeals-support/issues/3783#event-2054949783

Users want to be able to add tags from the keyboard. I feel this is a reasonable ask and pretty simple to add.

Maybe:

alt+shift+m for medical
alt+shift+p for procedural
alt+shift+o for other

Also add these shortcuts to the list of shortcuts modal in Reader to make them more discoverable.

### PR DESCRIPTION
**Pull request by Jeremy1026**
Resolves #8519 

### Description
Adds keyboard shortcuts in Reader for adding and removing category labels Procedural (Alt+Shift+P), Medical (Alt+Shift+M), and Other Evidence (Alt+Shift+O).

### Acceptance Criteria
- [ ] Code compiles correctly
- [ ] New shortcuts work
- [ ] Existing shortcuts continue to work

### Testing Plan
1. Go to reader view (ex: http://localhost:3000/reader/appeal/2226048/documents/21)
2. Send the new keyboard shortcuts and ensure the categories are toggled

### User Facing Changes
 - [ ] Screenshots of UI changes added to PR & Original Issue

image

### CODE REVIEW

File: client/app/reader/PdfKeyboardInfo.jsx [hunk 1]
```diff
@@ -73,3 +73,19 @@ export const documentsColumns = [{ header: 'Navigate reader',
...
+export const categoryColumns = [{ header: 'Add/Remove categories',
+  valueName: 'categoryInstruction',
+  align: 'left' },
+{ header: 'Shortcut',
+  valueName: 'shortcut',
+  align: 'left' }];
+
+export const categoryInstructions = [
+  { categoryInstruction: 'Add/Remove Medical',
+    shortcut: <span><code><code>alt</code> + <code>shift</code></code> + <code>m</code></span> },
```
Comment (n1zyy) [inline client/app/reader/PdfKeyboardInfo.jsx:86]:
```suggestion
    shortcut: <span><code>alt</code> + <code>shift</code> + <code>m</code></span> },
```

These ended up with nested `<code>` tags somehow, which isn't consistent with others (see, e.g., `searchInstructions` above

The other two instructions below are the same way.

**Review by n1zyy — COMMENTED** [review]
This all looks reasonable to me, but I'm not a JS guru. One small note around some extra `<code>` tags.

File: client/app/reader/PdfKeyboardInfo.jsx (hunk 1 above)
Comment (Jeremy1026) [reply client/app/reader/PdfKeyboardInfo.jsx]: Good catch, updated.

**Review by eileen-nava — APPROVED** [review]
I tested locally and the keyboard shortcuts work well for me. 👍

**Pull request merged**